modules/ocf_mirrors/manifests/monitoring.pp

```diff
@@ -8,15 +8,15 @@
     $upstream_protocol = 'http',
     $ensure = 'present',
   ) {
-
+  $local_url = "http://mirrors.ocf.berkeley.edu${local_path}/dists/${dist_to_check}/Release"
+  $upstream_url = "${upstream_protocol}://${upstream_host}${upstream_path}/dists/${dist_to_check}/Release"
   if $ensure == 'present' {
     file { "${project_path}/health":
-      content => template("ocf_mirrors/monitoring/${type}-health.erb"),
```

Comment:
? not using type param anymore?

seems like we need to preserve that. this was originally written such that we could add different types of healthchecks.

Reply:
Yeah. I'll add the type back to the file name. Maybe a healthchecks folder in scripts is a good addition for the future? Right now the only healthchecks running are ftpsync checks.